src/SeleniumLibrary/keywords/window.py

```diff
@@ -215,12 +215,12 @@ def set_window_size(self, width, height, inner=False):
             height_offset = height - inner_height
             window_width = width + width_offset
             window_height = height + height_offset
-            res = self.driver.set_window_size(window_width, window_height)
+            size = self.driver.set_window_size(window_width, window_height)
```

Comment:
If I'm not wrong... this does not contain the size, but just the execution status